src/weights.jl

```diff
@@ -96,8 +96,8 @@ end
 Construct an `AnalyticWeights` vector with weight values `vs`.
 A precomputed sum may be provided as `wsum`.
 
-Analytic weights describe a non-random relative importance (usually between 0 and 1)
-for each observation. These weights may also be referred to as reliability weights,
+Analytic weights represent the inverse of the variance for each case.
+These weights may also be referred to as reliability weights,
 precision weights or inverse variance weights. These are typically used when the observations
```

Comment:
```suggestion
precision weights or regression weights. These are typically used when the observations
```

Reply:
Are you sure the expression "regression weights" is used commonly? I'm afraid it could confuse users, as you can use any kind of weights in regression (e.g. Stata supports all three types).

Reply:
I think SAS uses that term for them, but I don't know how common it is elsewhere. I'm also fine with just removing the redundant `inverse variance` and only listing two other common names (e.g., `reliability weights`, `precision weights`).

https://blogs.sas.com/content/iml/2017/10/02/weight-variables-in-statistics-sas.html

Reply:
OK, let's do that.